### DISCUSSION TITLE
Multiple OIDC providers support

### DISCUSSION THREAD

**Discussion by pierreblais** [coder/coder#12778] [Feature Request]
Hi,

I would like to know if it is possible to connect multiple OIDC providers to manage authentication or if there are plans to support that features ? As I experienced, it's look like that is not possible 🙃

Thanks

**Comment by ammario** [comment]
It's an important feature we'll likely build in the next year to support larger deployments. Could you share more details about how would you use multiple providers and if there's a suitable work-around today?

**Comment by phorcys420** [comment]
I would like to mention that even if it is not a built-in feature yet (as per @ammario's comment), a good workaround can be to use an OIDC provider that has support for federating multiple from SSO sources (e.g [Authentik](https://goauthentik.io)/[Keycloak](https://www.keycloak.org)/etc).
